### PR TITLE
Mepts 859 update tpt initiation list

### PR DESCRIPTION
**Pull request by cfaife**

### CODE REVIEW

**Comment by ItsMurumba** [comment]
@cfaife  Could you fix conficts.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TPT_InitiationQueries.java [hunk 1]
```diff
@@ -1237,7 +1237,7 @@ public static String getTPTInitiationTOTALS(
     valuesMap.put("1", hivCareProgram);
 
     String query =
-        "  SELECT (SELECT name FROM location WHERE location_id = :location) AS location, COUNT(TPT.person_id) AS total, CONCAT(:startDate,' Até ',:endDate) AS period "
+        "  SELECT (SELECT name FROM location WHERE location_id = :location) AS location, COUNT(TPT.person_id) AS total, CONCAT(DATE_FORMAT(:startDate, '%Y %M %d'),' Até ',DATE_FORMAT(:endDate, '%Y %M %d')) AS period "
```
Comment (ItsMurumba) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TPT_InitiationQueries.java]: @cfaife  I see a difference in date format that Pinkie Had requested. (**dd-mm-yyyy** )

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TPT_InitiationQueries.java [hunk 2]
```diff
@@ -1237,7 +1237,7 @@ public static String getTPTInitiationTOTALS(
     valuesMap.put("1", hivCareProgram);
 
     String query =
-        "  SELECT (SELECT name FROM location WHERE location_id = :location) AS location, COUNT(TPT.person_id) AS total, CONCAT(DATE_FORMAT(:startDate, '%Y %M %d'),' Até ',DATE_FORMAT(:endDate, '%Y %M %d')) AS period "
+        "  SELECT (SELECT name FROM location WHERE location_id = :location) AS location, COUNT(TPT.person_id) AS total, CONCAT(DATE_FORMAT(:startDate, '%Y-%M-%d'),' Até ',DATE_FORMAT(:endDate, '%Y-%M-%d')) AS period "
```
Comment (ItsMurumba) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TPT_InitiationQueries.java]: Same as my previous comment.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupIntensiveMonitoringReport.java [hunk 3]
```diff
@@ -16,9 +16,8 @@
 import org.openmrs.module.reporting.report.ReportDesign;
 import org.openmrs.module.reporting.report.definition.ReportDefinition;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
 
-@Component
+@Deprecated
```
Comment (ItsMurumba) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupIntensiveMonitoringReport.java]: Are we deprecating the class just for testing purpose? @cfaife

**Review by ItsMurumba — COMMENTED** [review]
Left some comments. Everything else seems okay for me. @cfaife

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**